.github/workflows/ci-workflow.yml

```diff
@@ -160,22 +159,19 @@ jobs:
           ${{ env.MAVEN_COMPILE_COMMAND }}
       - name: Maven Verify
         env:
-          NIFI_CI_LOCALE: >-
-            -Duser.language=en
-            -Duser.country=AU
-          SUREFIRE_OPTS: >-
```

Comment:
It might be possible to retain these args (thus allowing use of JaCoCo in other Actions in future if desired, without needing to change the `mvn` args, by using something like:

```xml
                    <plugin>
                        <groupId>org.apache.maven.plugins</groupId>
                        <artifactId>maven-surefire-plugin</artifactId>
                        <configuration>
                            <argLine>@{jaCoCoArgLine} -Xmx1024m</argLine>
                        </configuration>
                    </plugin>
```

within `<pluginManagement/>`, and (within `<build/>`):

```xml
                    <plugin>
                        <groupId>org.jacoco</groupId>
                        <artifactId>jacoco-maven-plugin</artifactId>
                        <executions>
                            <execution>
                                <id>default-prepare-agent</id>
                                <goals>
                                    <goal>prepare-agent</goal>
                                </goals>
                                <configuration>
                                    <propertyName>jaCoCoArgLine</propertyName>
                                </configuration>
                            </execution>
                        </executions>
                    </plugin>
```

Reply:
I considered similar options as described in the [prepare-agent](https://www.jacoco.org/jacoco/trunk/doc/prepare-agent-mojo.html) goal documentation, but part of the difficulty is that the build workflow was setting a System property, which JaCoCo also needs to set. We could change the System property in the build workflow to a different name, and then use that name for late property evaluation in the maven-surefire-plugin configuration, but that would have involved changes all of the build profiles. I kept the scope of changes narrow in this case, but we could revisit in the future if we need to introduce other arguments.